### PR TITLE
NMS-9718: Unexpected errors while generating event definitions from TRAP-TYPE without a DESCRIPTION field

### PR DESCRIPTION
**Pull request by agalue**
Unexpected errors while generating event definitions from TRAP-TYPE without a DESCRIPTION field

* JIRA: http://issues.opennms.org/browse/NMS-9718

Our [continuous integration system](http://bamboo.internal.opennms.com:8085) will test and verify your changes.

### CODE REVIEW

File: features/mib-compiler/src/main/java/org/opennms/features/mibcompiler/services/JsmiMibParser.java [hunk 1]
```diff
@@ -157,7 +157,7 @@ public boolean parseMib(File mibFile) {
             try {
                 mib = parser.parse();
             } catch (Exception e) {
-                LOG.error("Can't compile {}", mibFile, e);
+                LOG.error("Can't compile " + mibFile, e);
```
Comment (j-white) [inline features/mib-compiler/src/main/java/org/opennms/features/mibcompiler/services/JsmiMibParser.java]: Why change these? Both of these end up printing the same message.

```
Exception e = new Exception();
String param = "p1";
LOG.error("error: {}", param, e);
LOG.error("error: " + param, e);
```

```
2017-10-30 09:54:19,207 ERROR [main] org.opennms.netmgt.snmp.LogTest - error: p1
java.lang.Exception
	at org.opennms.netmgt.snmp.LogTest.doIt(LogTest.java:40)
	...
2017-10-30 09:54:19,213 ERROR [main] org.opennms.netmgt.snmp.LogTest - error: p1
java.lang.Exception
	at org.opennms.netmgt.snmp.LogTest.doIt(LogTest.java:40)
	...
```

Comment (agalue) [reply features/mib-compiler/src/main/java/org/opennms/features/mibcompiler/services/JsmiMibParser.java]: I was unsure that if I add placeholders, and then the throwable at the end, I can have the full exception trace besides my custom message (based on the docs). So, are you saying that will happen ? If so, I can rollback those particular changes.

Comment (j-white) [reply features/mib-compiler/src/main/java/org/opennms/features/mibcompiler/services/JsmiMibParser.java]: https://www.slf4j.org/faq.html#paramException

**Comment by agalue** [comment]
Done! let me know if there is something else I should change.

**Pull request merged**